tests/runtimes/local/agent_runtime_test.py

```diff
@@ -83,6 +83,7 @@ def testCreateAgentService_whenAgentDefAndAgentSettingsAreNotEmpty_serviceCreate
 
     # assert arguments were overridden by the agent settings.
     assert kwargs["resources"]["Limits"]["MemoryBytes"] == 700000
+    assert len(kwargs["name"]) < 63
```

Comment:
Can you assert on the name value. We need to ensure we don't end up with the same name.